### PR TITLE
Adding  TS typings and fixing one mistype error in README.

### PR DESCRIPTION
**Pull request by Fazendaaa**
Hey, @theoephraim . Thanks for this awesome package!

Due to Hacktoberfest 2018, I'm contributing to some packages that I use; yours is one of those.

TypeScript typings help a lot of us who use it due to it's integration to TSLint and some more strict checkings, needed to be performed in some cases in the used packages as well; that's why I've decided to share the ones that I've made, making it more accessible to all that programs in this environment :)

* [x] No tests were implemented -- since I didn't modify any of your code
* [x] Linked typings file in your package.json
* [x] Documentation modified to fix one mistype error and add some of the callback definitions.

:)

### CODE REVIEW

**Comment by mastef** [comment]
@Fazendaaa wouldn't you need to add a test-case to check if the types still apply? In case there's an architecture change - e.g a parameter removed or added in a new commit

**Comment by ukaric** [comment]
This is good merge candidate, tests can be added on the fly.

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+         */
+        public setAuthToken(id: string): void;
+
+        /**
+         * Get information about the spreadsheet
+         * 
+         * @param callback Default callback containing requested the info
+         */
+        public getInfo(callback: (err: Error, info: GetInfo) => void): void;
+
```
Comment (arielshaqed) [inline index.d.ts]:
```suggestion
        /**
         * Get an array of row objects from the sheet.
         * 
         * @param worksheet_id The index of the sheet to read from (index starts at 1)
         * @param callback Default callback containing array of SpreadsheetRow object
         */
        public getRows(worksheet_id: number, callback: (err: Error, rows: SpreadsheetRow[]) => void): void;
```

**Comment by theoephraim** [comment]
Just a heads up - a full rewrite is imminent, because google is deprecating their v3 api on March 2nd. I'm hoping to release it in the next week or so.

**Comment by theoephraim** [comment]
FYI - I've just a published a new version of this module which is a complete rewrite using Google's V4 Sheets API (v3 is being deprecated on March 3, 2020). **YOU MUST MIGRATE IF YOU ARE STILL USING THIS PACKAGE!**

**Docs available here - https://theoephraim.github.io/node-google-spreadsheet**

If you want to do new TS typings, I'd be grateful :)

**Comment by Fazendaaa** [comment]
@theoephraim, I will take a look at it :)

**Pull request closed without merging**